src/services/email/serverless.yml

```diff
@@ -26,13 +31,22 @@ provider:
           Action:
             - ses:ListIdentities
             - ses:ListConfigurationSets
-            - ses:SendEmail
+            - ses:SendTemplatedEmail
           Resource: "*"
         - Effect: Allow
           Action:
             - sns:Subscribe
             - sns:Publish
           Resource: "*"
+        - Effect: Allow
+          Action:
+            - ec2:CreateNetworkInterface
+            - ec2:DescribeNetworkInterfaces
+            - ec2:DescribeVpcs
+            - ec2:DeleteNetworkInterface
+            - ec2:DescribeSubnets
+            - ec2:DescribeSecurityGroups
+          Resource: "*"
```

Comment:
heh... I have no idea.  They most likely got added because a deployment failed from a lack of permission, but that could've been early on when I was investigating something else.  I will check if they are needed anymore.